nixos/lib/systemd-lib.nix

```diff
@@ -382,6 +391,22 @@ in rec {
         '';
     };
 
+  initrdServiceToUnit = name: def:
```

Comment:
This only differs from the `serviceToUnit` definition above in not setting the `X-ReloadIfChanged/X-RestartIfChanged` fields, no? Is there any harm in having these fields around?

 - If there is, could the `initrdServiceToUnit` be a function moved to `nixos/modules/system/boot/systemd/initrd.nix`?
 - If there is not, could we just use `serviceToUnit` there?

Reply:
I pushed e3083decc4dea10e34ac08b705ea74808cefa524, which removes initrdServiceToUnit and replaces it just with serviceToUnit, which still seems to work. PTAL.